mw-webapp/src/logic/wayPage/wayStatistics/statisticWidget/StatisticWidget.module.scss

```diff
@@ -8,6 +8,11 @@
 }
 
 .titleCheckbox {
+  position: relative;
   width: 100%;
   justify-content: space-between;
 }
+
+.titleCheckboxIndex {
+  z-index: $checkboxZIndex;
```

Comment:
Are you sure we don't have other options to solve the problem with clickable checkboxes?